sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/fluent/models/AppServicePlanProperties.java

```diff
@@ -47,6 +47,12 @@ public final class AppServicePlanProperties {
     @JsonProperty(value = "maximumNumberOfWorkers", access = JsonProperty.Access.WRITE_ONLY)
     private Integer maximumNumberOfWorkers;
 
+    /*
+     * The number of instances that are assigned to this App Service plan.
+     */
+    @JsonProperty(value = "numberOfWorkers", access = JsonProperty.Access.WRITE_ONLY)
+    private Integer numberOfWorkers;
```

Comment:
Please note that this class is generated from swagger https://github.com/Azure/azure-rest-api-specs/blob/main/specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/CommonDefinitions.json#L172

In their latest version of "AppServicePlan.properties" definition, there is no "numberOfWorkers" property. Without service defines it, SDK does not know whether service will always honor this property.

Note that this "numberOfWorkers" exists in SiteConfig.

Reply:
Thanks @weidongxu-microsoft.  Sounds like I should open a PR in the azure-rest-api-specs repo for the swagger file given these classes in my PR are generated from it.  The `numberOfWorkers` in the SiteConfig that I'm seeing only applies to the app service, not the app service plan.  I want to get at the `numberOfWorkers` property of the app service plan.

That property stems from the app service plan's configurable number of instances:
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/20073455/170691101-43d4f758-9af5-4da7-8ed0-52e5f500acaf.png">

This corresponds to the `numberOfWorkers` of the app service plan as shown in the json view of the app service plan:
![image](https://user-images.githubusercontent.com/20073455/170691596-4309fac0-4fa2-4034-a708-9e62d9661ba8.png)

Reply:
Sure. If you have created issue on azure-rest-api-specs repo, you can link it here.

If there is no response on azure-rest-api-specs issue, I should be able to escalate it internally. Just @ me on that issue.

Reply:
Thanks @weidongxu-microsoft , the issue from the azure-rest-api-specs repo is here:
https://github.com/Azure/azure-rest-api-specs/issues/7990